doc/release-notes/5.3-release-notes.md

```diff
@@ -0,0 +1,152 @@
+```
+
+## Notes for Tool Developers and Integrators
+
+### Experimental Auxiliary File Support
+
+Experimental endpoints have been added to allow auxiliary files to be added to datafiles. These auxiliary files can be deposited and accessed via API. Later releases will include options for accessing these files through the UI. For more information, see the Auxiliary File Support section of the [Developer Guide](https://guides.dataverse.org/en/5.3/developers/).
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.3 Milestone](https://github.com/IQSS/dataverse/milestone/93?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.3/installation/).
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
+
+1\. Upgrade to Payara 5.2020.6.
+
+Instructions how to update can be found in the
+[Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
+
+2\. Undeploy the previous version.
+
+- `<payara install path>/bin/asadmin list-applications`
+- `<payara install path>/bin/asadmin undeploy dataverse<-version>`
+
+(where `<payara install path>` is where Payara 5 is installed, for example: `/usr/local/payara5`)
+
+3\. Update your database connection.
+
+Please configure your connection details, replacing all the `${DB_...}`.
+(If you are using a PostgreSQL server on `localhost:5432`, you can omit `dataverse.db.host` and `dataverse.db.port`.)
+
+- `<payara install path>/asadmin create-system-properties "dataverse.db.user=${DB_USER}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.host=${DB_HOST}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.port=${DB_PORT}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.name=${DB_NAME}"`
+- `echo "AS_ADMIN_ALIASPASSWORD=${DB_PASS}" > /tmp/password.txt`
+
+- `<payara install path>/asadmin create-password-alias --passwordfile /tmp/password.txt dataverse.db.password`
+- `rm /tmp/password.txt`
+
+4\. Reset the EJB timer database back to default:
+
+- `<payara install path>/bin/asadmin set configs.config.server-config.ejb-container.ejb-timer-service.timer-datasource=jdbc/__TimerPool`
+
+5\. Delete the old password alias and DB pool:
+
+- `<payara install path>/asadmin delete-jdbc-connection-pool --cascade=true dvnDbPool`
+- `<payara install path>/asadmin delete-password-alias db_password_alias`
```

Comment:
these are missing /bin/ before asadmin

Reply:
added in https://github.com/IQSS/dataverse/commit/f82f970aa6e1706dcf631eebcb498c159f6ffc96